src/features/update/update-model.ts

```diff
@@ -203,6 +203,12 @@ export class UpdateModelCommand extends Command {
         } else {
             return newRoot;
         }
+
+        function containsSome(root: SModelRoot, element: SChildElement): boolean {
+            const test = (element: SChildElement) => root.index.getById(element.id) !== undefined;
+            const find = (elements: readonly SChildElement[]): boolean => elements.some(element => test(element) || find(element.children));
+            return find([element]);
+        }
```

Comment:
It's unusual in our codebase so far to have such functions in a local scope. This could be moved to `smodel-utils.ts`?

Reply:
Yeah, I had mixed feelings about placing the function there: reusability vs. high cohesion. I will move it to smodel-utils.